packages/composites/dydx-rewards/test/integration/ipfs-data.test.ts

```diff
@@ -9,6 +9,8 @@ import {
 import mockRewards from '../mock-data/rewards.json'
 import nock from 'nock'
 import { mockIpfsResponseSuccess } from './fixtures'
+import { AdapterRequest, Execute } from '@chainlink/ea-bootstrap'
```

Comment:
```suggestion
import type { AdapterRequest, Execute } from '@chainlink/ea-bootstrap'
```
For type only imports it is nice to mark them that way for the compiler